deployment-templates/compute-engine/server/forseti-instance-server.py.schema

```diff
@@ -45,6 +45,10 @@ properties:
     type: string
     desription: The GCE instance type.
 
+  region:
```

Comment:
Why do we need this new `region` section for VPC?

Reply:
Subnets only have a concept of region, not zone. So the subnet takes the form:

/projects/{}/regions/{}/subnetworks/{}

Does that make sense?

Reply:
Ah, I see.  Before we were using a cloudsql region.  So, this is better for sure.